### PR TITLE
add mysql deadlocks chart

### PR DESCRIPTION
**Pull request by kam1kaze**
##### Summary
Add innodb deadlock chart to mysql.

##### Component Name
collectors

### CODE REVIEW

File: collectors/python.d.plugin/mysql/mysql.chart.py [hunk 1]
```diff
@@ -382,6 +384,13 @@ def slave_running(value):
             ['Innodb_row_lock_current_waits', 'current_waits', 'absolute']
         ]
     },
+    'innodb_deadlocks': {
+        'options': [None, 'InnoDB Deadlocks', 'operations', 'innodb',
```
Comment (ilyam8) [inline collectors/python.d.plugin/mysql/mysql.chart.py]: units is `operations/s`

**Comment by kam1kaze** [comment]
@ilyam8 fixed

File: collectors/python.d.plugin/mysql/mysql.chart.py [hunk 2]
```diff
@@ -357,7 +359,7 @@ def slave_running(value):
         ]
     },
     'innodb_os_log': {
-        'options': [None, 'InnoDB OS Log Pending Operations', 'operations', 'innodb', 'mysql.innodb_os_log', 'line'],
+        'options': [None, 'InnoDB OS Log Pending Operations', 'operations/s', 'innodb', 'mysql.innodb_os_log', 'line'],
         'lines': [
             ['Innodb_os_log_pending_fsyncs', 'fsyncs', 'absolute'],
             ['Innodb_os_log_pending_writes', 'writes', 'absolute', -1, 1],
```
Comment (ilyam8) [inline collectors/python.d.plugin/mysql/mysql.chart.py]: absolutes are not `/s`, incrementals are. (netdata does interpolation only for incrementals)

**Comment by kam1kaze** [comment]
@ilyam8  got it, roll back such changes

**Comment by cakrit** [comment]
Thanks again @kam1kaze ! :)

**Pull request merged**